requirements.txt

```diff
@@ -1,25 +1,57 @@
 certifi==2024.7.4
```

Comment:
the file is now re-generated with `tox -e upgrade`.
Mose important change: `hansken_extraction_plugin==0.8.1+master.77.3b0112ba` to `hansken_extraction_plugin==0.8.1`